GeometryOpsCore/src/types.jl

```diff
@@ -51,6 +51,8 @@ end
 
 A spherical manifold means that the geometry is on the 3-sphere (but is represented by 2-D longitude and latitude).  
 
+By default, the radius is defined as the mean radius of the Earth, `6371008.8 m`.
```

Comment:
Can we make global named constants for all the numbers in these files, and both insert them in dics and use them for keywords?

Maybe in a different PR, just commenting because this text is added here